src/familytoolswap_backend/routes.py

```diff
@@ -93,9 +93,23 @@ def twotables():
 
 #@app.route("/customer/history/year/<year>", methods=["GET"])
 
-@app.route("/tool_search/<desc>")
-def tool_search(desc):
+@app.route("/tool_search/<description>/<type>")
+def tool_search(description, type):
+	# tool_search_sql = "SELECT id, name, description FROM tool_inventory JOIN owner ON tool_inventory.id = owner.owner_id WHERE description = '" + description +"'"
+	# tool_search_sql = f"SELECT id, name, description FROM tool_inventory JOIN owner ON tool_inventory.id = owner.owner_id WHERE description = '{ description }'"
+	tool_search_sql = "SELECT id, name, description, type FROM tool_inventory JOIN owner ON tool_inventory.id = owner.owner_id WHERE description = %s and type = %s"
 	cursor = postgres.cursor()
-	cursor.execute("SELECT id, name, description FROM tool_inventory JOIN owner ON tool_inventory.id = owner.owner_id WHERE description = '" + desc +"'")
+	cursor.execute(tool_search_sql, [description, type])
+	jaysonresults=[]
+	for id, name, description, type in cursor.fetchall():
+		print(description)
```

Comment:
Consider removing this print statement we used for debugging if you feel you don't need it any more. (This one and line 84)